.travis.yml

```diff
@@ -6,11 +6,15 @@ python:
 sudo: false
 cache: pip
 
+
 addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml  # because pip installation is slow
+      - python-simplejson
+      - python-serial
+      - python-yaml
```

Comment:
Grazie della segnalazione, rimetto come in origine.

Mi chiedevo come mai il file travis del progetto italiano non è uguale al template dei quality tools:
https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml